packages/lib-user/src/utils/createPanoptesMembership.js

```diff
@@ -0,0 +1,24 @@
+import { panoptes } from '@zooniverse/panoptes-js'
+import auth from 'panoptes-client/lib/auth'
+
+export async function createPanoptesMembership({
+  groupId,
+  joinToken,
+  userId
+}) {
+  const token = await auth.checkBearerToken()
+  const authorization = `Bearer ${token}`
+  if (!token) return null
+
+  const response = await panoptes.post('/memberships',
+    { memberships: {
+      join_token: joinToken,
+      links: {
+        user: userId,
+        user_group: groupId
+      }
+    }},
+    { authorization }
+  )
+  return response
```

Comment:
Just to double check - do the `utils` need some basic error handling? For instance, `getBearerToken()` util has a try/catch, but the other functions do not, and they all return `response` from `panoptes` regardless. There may be some error handling already in `panoptes` from `@zooniverse/panoptes-js`, but good to double check at this PR!

Reply:
👍 Added console.error's to related resource utils. Still throwing the error for consuming function/component to do as necessary with, as opposed to getBearerToken which I think expectedly captures the error, I think with these resource utils we want the error to be thrown, so slight difference with throw/capture error intended (though let me know if mistaken in understanding).